setup/plugins/installDatabase/BootstrapCivi.civi-setup.php

```diff
@@ -2,7 +2,15 @@
 /**
  * @file
  *
- * Perform the first system bootstrap.
+ * Perform the full bootstrap.
+ *
+ * GOAL: All the standard services (database, DAOs, translations, settings, etc) should be loaded.
+ *
+ * MECHANICS: This is basically calls `CRM_Core_Config::singleton(TRUE,TRUE)`.
```

Comment:
Grammar-police.

Reply:
A life of poor grammatical choices will put you behind 💈💈💈💈